{
  Class c=Class.forName(getExceptionClassName());
  Constructor constructor=c.getConstructor((Class[])null);
  Exception e=(Exception)constructor.newInstance((Object[])null);
  Assert.assertNotNull("Exception should be created",e);
  Assert.assertNull(e.getMessage());
  Assert.assertNull(e.getCause());
}
